pkg/reconciler/ingress/config/istio.go

```diff
@@ -60,6 +63,11 @@ func defaultLocalGateways() []Gateway {
 		Name:      ClusterLocalGateway,
 		ServiceURL: fmt.Sprintf(ClusterLocalGateway+".istio-system.svc.%s",
 			network.GetClusterDomainName()),
+	}, {
+		Namespace: system.Namespace(),
+		Name:      KnativeLocalGateway,
+		ServiceURL: fmt.Sprintf(KnativeLocalGateway+".knative-serving.svc.%s",
```

Comment:
This should be using `system.Namespace()`

Reply:
In fact, we have a helper for synthesizing service hostnames that this should be using too in place of `GetClusterDomainName()`